database-commons/src/main/java/io/cdap/plugin/db/action/AbstractDBArgumentSetter.java

```diff
@@ -133,9 +136,11 @@ private void processArguments(Class<? extends Driver> driverClass,
    */
   private void setArguments(ResultSet resultSet, FailureCollector failureCollector,
                             SettableArguments arguments) throws SQLException {
-    String[] columns = config.getArgumentsColumns().split(",");
-    for (String column : columns) {
-      arguments.set(column, resultSet.getString(column));
-    }
+    Failsafe.with(RetryPolicyUtil.RETRY_POLICY).run(() -> {
+      String[] columns = config.getArgumentsColumns().split(",");
+      for (String column : columns) {
+        arguments.set(column, resultSet.getString(column));
+      }
+    });
```

Comment:
Is there any call to the backend taking place here that can cause `SQLTransientConnectionException`? If not, retry is not needed here.